charts/tfy-inferentia-operator/templates/neuron-device-plugin.yaml

```diff
@@ -42,6 +42,10 @@ kind: ServiceAccount
 metadata:
   name: {{ .Release.Name }}-neuron-device-plugin
   namespace: {{ .Release.Namespace }}
+{{- if .Values.imagePullSecrets }}
```

Comment:
If we have two different keys for imagePullSecrets, we should perform the override before using the service specific one here or in the pod spec. We don't really need to inject at the service account level tbh

Reply:
This is a good example https://github.com/argoproj/argo-helm/blob/d61432be3c1e460e7037462b6b5d5f8f1bdcc6d7/charts/argo-cd/templates/argocd-server/deployment.yaml#L47

Reply:
Thanks, I made the changes